### PR TITLE
feat: update collapsible sidebar implementation

### PR DESCRIPTION
**Pull request by thisisamir98**
## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx [hunk 1]
```diff
@@ -102,12 +101,14 @@ const Conversations: React.FC<ConversationsProps> = ({
...
   selfUser,
 }) => {
-  const [isSidebarCollapsed, setIsSidebarCollapsed] = useState(false);
+  const {isOpen: isSideBarOpen, toggleIsOpen: toggleSidebarIsOpen} = useSidebarStore();
   const [conversationsFilter, setConversationsFilter] = useState<string>('');
   const {activeCalls} = useKoSubscribableChildren(callState, ['activeCalls']);
   const {classifiedDomains} = useKoSubscribableChildren(teamState, ['classifiedDomains']);
   const {connectRequests} = useKoSubscribableChildren(userState, ['connectRequests']);
 
+  const isSidebarCollapsed = !isSideBarOpen;
```
Comment (przemvs) [inline src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx]: Do we need this ? I think You can use it directly ;)

File: src/style/list/conversations.less [hunk 2]
```diff
@@ -26,88 +26,67 @@
...
-    z-index: 1000;
-    display: flex;
-    overflow: auto;
-    width: 220px;
-    height: 100vh;
-    background: var(--sidebar-bg);
-    transform: translateX(0);
+    // &:hover {
+    //   opacity: 1;
+    // }
```
Comment (przemvs) [inline src/style/list/conversations.less]: This code is necessary? :)

File: src/style/list/conversations.less [hunk 3]
```diff
@@ -26,88 +26,67 @@
...
-    width: 16px;
-    height: 100%;
-    opacity: 0;
+  .conversations-sidebar-items {
+    display: flex;
+    overflow: hidden;
+    width: 220px;
+    height: 100vh;
+    flex-direction: column;
+    padding: 8px 8px;
```
Comment (przemvs) [inline src/style/list/conversations.less]:
```suggestion
    padding: 8px;
```

**Review by przemvs — APPROVED** [review]
LGTM, few things :)

File: src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx (hunk 1 above)
Comment (thisisamir98) [reply src/script/page/LeftSidebar/panels/Conversations/Conversations.tsx]: done in https://github.com/wireapp/wire-webapp/pull/17218/commits/0107c8d95be5ae866221d5bc3d19126a36294f9d

File: src/style/list/conversations.less (hunk 2 above)
Comment (thisisamir98) [reply src/style/list/conversations.less]: done in https://github.com/wireapp/wire-webapp/pull/17218/commits/f4815175f1892e35a677e224e60816ef13b9f8cc

File: src/style/list/conversations.less (hunk 3 above)
Comment (thisisamir98) [reply src/style/list/conversations.less]: done in https://github.com/wireapp/wire-webapp/pull/17218/commits/6f53dc318469e309320f09206670206f1ecf488f

**Pull request merged**